SPECS/dracut/add-livenet-download-service.patch

```diff
@@ -0,0 +1,147 @@
+From 2888695d6e1e03f727fe45d6e38fc71f7b5be123 Mon Sep 17 00:00:00 2001
```

Comment:
This patch means that the MIC PXE feature will require a minimum version of Azure Linux 3.0 to be able to function. In addition, you will be unlikely to be able to apply this patch to Azure Linux 2.0.

At a minimum, MIC should check for this patch in some way and error out if it isn't available. That will at least avoid the situation where it silently fails for the user.

Though, it might be good to investigate ways we could make MIC PXE feature independent of the Azure Linux version.

Reply:
Added logic to check the dracut version and if below a certain version, it:
- gives a warning that it will not generate the grub.cfg for pxe.
- gives an error if the user specifies the `--output-pxe-artifacts-dir`.